student/models.py

```diff
@@ -59,6 +59,7 @@ class year, major, friends, etc. An object is only created for the
     first_name = models.CharField(max_length=255, default='', null=True)
     last_name = models.CharField(max_length=255, default='', null=True)
     disabilities = models.NullBooleanField(null=True, default=False)
+    favorite_num = models.IntegerField(blank=True, null =True)
```

Comment:
This is looking great so far, just add 'favorite_num' to serializers.py and run migrations and it should work!